sources/packages/backend/apps/db-migrations/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Base Image
+FROM registry.access.redhat.com/ubi8/nodejs-16:1-5
+
+LABEL maintainer="BC GOV"
+
+# Copying package.json and package-lock.json
+COPY package*.json ./
+
+# Installing dependencies
+RUN npm ci --ignore-scripts
```

Comment:
I see that we are trying to limit the packages here to what has been present in package.json. Are we sure of no side effects?

Reply:
using npm i --ignore-scripts is entirely different from using npm ci --ignore-scripts
npm i will execute lifecycle scripts associated with those packages. These scripts can include pre-install, post-install, pre-publish, and other custom scripts defined by the package author. Using --ignore-scripts will entirely skip all such scripts.
npm ci is used to download dependencies using the package-lock.json, by default npm ci does not run any  lifecycle scripts. However for the sake of sonarcloud, we are adding --ignore-scripts in our Dockerfile.
I am sure this will not have any side effects.